src/components/Dropdown/Dropdown.tsx

```diff
@@ -1,25 +1,64 @@
 import Link from 'next/link';
-import { LOGOUT_URL, USER_PROFILE_URL } from '../../constants/url';
+import {
+    LOGOUT_URL,
+    MAIN_SITE_URL,
+    MY_SITE_URL,
+    USER_PROFILE_URL,
+} from '../../constants/url';
 import styles from '@/components/Dropdown/dropdown.module.scss';
 
+export const logout = () => {
+    fetch(LOGOUT_URL, {
+        method: 'GET',
+        credentials: 'include',
+    }).then(() => {
+        location.reload();
+    });
+};
+
 const Dropdown = () => {
-    const logout = () => {
-        fetch(LOGOUT_URL, {
-            method: 'GET',
-            credentials: 'include',
-        }).then(() => {
-            location.reload();
-        });
-    };
+    // // const logout = () => {
+    // //     fetch(LOGOUT_URL, {
+    // //         method: 'GET',
+    // //         credentials: 'include',
+    // //     }).then(() => {
+    // //         location.reload();
+    // //     });
+    // };
 
     return (
-        <div className={styles.dropdown}>
-            <ul className={styles.dropdownList}>
+        <div className={styles.dropdown} data-testid="dropdown">
+            <ul className={styles.dropdownList} data-testid="options">
```

Comment:
instead of testid can we not define a role for and use the role to test the particular component

Reply:
This is not wrong either @Pratiyushkumar we can use this